index.html

```diff
@@ -40,6 +41,188 @@ <h3 class="subject_text">I'm a Software Developer</h3>
         </ul>
         <div class="illustration"></div>
     </section>
+    <section class="portfolio">
+        <div class="Portfolio_title_container">
+            <h2 class="portfolio_title">My Recent Works</h2>
+        </div>
+        <div class="separation">
+            <hr>
+        </div>
+        <div class="feature-project_preview_image"></div>
+        <div class="feature-project_title">
+            <h3>Architecture</h3>
+        </div>
+        <div class="feature-project_description_container">
+            <h4 class="feature-project_description">A daily selection of privately personalized reads; no 
+            accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, 
+            when an unknown printer took a standard dummy text.</h4>
```

Comment:
Kindly, I suggest you use the `p` tag instead of the `h4` to follow HTML best practices. 😄